apps/test-viewer/src/components/Viewer.tsx

```diff
@@ -9,10 +9,12 @@ import { Viewer as WebViewer, ViewerPerformance } from "@itwin/web-viewer-react"
 import { history } from "../history";
 import { getUiProvidersConfig } from "../UiProvidersConfig";
 import { useAuthorizationContext } from "./Authorization";
+import { BreakdownTrees } from "@itwin/breakdown-trees-react";
 
 const uiConfig = getUiProvidersConfig();
 
 async function onIModelAppInit() {
+  await BreakdownTrees.initialize();
```

Comment:
I think the plan was to initialize all components within their widget's initialization function rather than at the app level. E.g. in `UiProvidersConfig.tsx` change the tree widget's config to:
```tsx
  [
    "tree-widget",
    {
      initialize: async () => {
        await TreeWidget.initialize();
        await BreakdownTrees.initialize();
      },
      createUiItemsProvider: () => new TreeWidgetUiItemsProvider({
        additionalTrees: [getSpatialTree()]
      }),
    }
  ],
```

Adding @saskliutas for his opinion.

Reply:
changed